### PR TITLE
docs : added login-signup pages only JSX

### PR DESCRIPTION
**Pull request by omkarpatke**
# Added Login and SignUp pages only JSX

## Screenshots
quiz-login-page
quiz-signup-page

**Checks**
- Login.js
- SignUp.js

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import './App.css';
 import Home from './components/HomePage/Home';
 import { Routes, Route } from "react-router-dom";
+import Login from './components/Login/Login';
+import SignUp from './components/SignUp/SignUp';
 import Navbar from './components/Navbar/Navbar';
```
Comment (yaswanthmyneni) [inline src/App.js:6]: suggestion: create index file in components folder and export all components from it

File: src/components/SignUp/SignUp.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  return (
+    <>
+    <div className="login-container">
+        <h3 className='login-container-heading'>Account Information</h3>
+        <div className="login-card">
+            <h2>Sign Up</h2>
+            <form className="logIn-form">
+                <label htmlFor="full-name" aria-required="true">Full Name<span>*</span></label>
+                <input type="text" name="ful-name" id="full-name"/>
+
```
Comment (yaswanthmyneni) [inline src/components/SignUp/SignUp.jsx:15]: changes: in name there is a typo 'ful-name'

**Review by yaswanthmyneni — APPROVED** [review]
Everything is good to merge.

File: src/components/SignUp/SignUp.jsx (hunk 2 above)
Comment (omkarpatke) [reply src/components/SignUp/SignUp.jsx:15]: thanks, will correct it.

File: src/components/Login/Login.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Login.css';
+import { Link } from 'react-router-dom';
+
+export default function Login() {
+  return (
+    <>
+    <div className="login-container">
+        <h3 className="login-heading">Account Information</h3>
```
Comment (manish-GH) [inline src/components/Login/Login.js:9]: I don't see this line on the screenshot, is this required?

File: src/components/Login/Login.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+                <label htmlFor='login-password' aria-required="true">Password<span>*</span></label>
+                <input type="password" name="login-password" id="login-password"/>
+
+                <button className="forgot-password-link">Forgot your password?</button>
+
+                <button className="login-btn" type="submit"> Sign In </button>
+            </form>
+            <h2>OR</h2>
+            <button className="signIn-with-google-btn"><i className="lni lni-google"></i> <span>Sign in with google</span></button>
```
Comment (manish-GH) [inline src/components/Login/Login.js]: I would recommend you to shift google signIn to the bottom just like the signUp form

File: src/components/SignUp/SignUp.jsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './SignUp.css';
+import { Link } from 'react-router-dom';
+
+export default function SignUp() {
+  return (
+    <>
+    <div className="login-container">
+        <h3 className='login-container-heading'>Account Information</h3>
```
Comment (manish-GH) [inline src/components/SignUp/SignUp.jsx:9]: Is this required?

File: src/components/Login/Login.css [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+    border: none;
+    font-size: 1rem;
+    background-color: var(--google-bg-color);
+    color: var(--white-color);
+    transition: .8s all ease-in-out;
+}
+
+
+
+.signIn-with-google-btn:hover{
```
Comment (manish-GH) [inline src/components/Login/Login.css:100]: The text and background color contrast seem a bit off. Try different colors

**Review by manish-GH — APPROVED** [review]
The code and the names used are good. Look into these changes and then you are good to merge

File: src/components/Login/Login.js (hunk 3 above)
Comment (omkarpatke) [reply src/components/Login/Login.js:9]: Done ,with the changes

File: src/components/Login/Login.css (hunk 6 above)
Comment (omkarpatke) [reply src/components/Login/Login.css:100]: ok

**Pull request merged**